src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -113,6 +116,25 @@ data class MigrateEvidence(
   val reporter: String,
 )
 
+@Schema(description = "punishment / sanction")
+data class MigratePunishment(
+  @Schema(description = "sanction code")
+  val sanctionCode: String,
+  @Schema(description = "sanction status")
+  val sanctionStatus: String,
+  @Schema(description = "sanction sequence")
+  val sanctionSeq: Long,
+  @Schema(description = "effective date")
+  val effectiveDate: LocalDateTime,
+  @Schema(description = "comment text")
+  val comment: String?,
```

Comment:
currently we use this to store the OTHER types (ie privilege type etc).  For migration, we will put the comment text in our punishment comments object, so it will be linked to the report itself, and not the specific sanction.   likely will add the sanction type in the comment as well so its clear